packages/rancher-logging/generated-changes/patch/values.yaml.patch

```diff
@@ -26,16 +26,17 @@
  
  affinity: {}
  
-@@ -50,6 +55,8 @@
+@@ -50,6 +55,9 @@
      # Labels to query http service
      labels: {}
  
 +# These "rbac" settings match the upstream defaults. For only using psp in the overlay files, which
-+# include the default Logging CRs created, see the "global.psp" setting.
++# include the default Logging CRs created, see the "global.psp" setting. To enable psp for the entire
++# chart, enable both "rbac.psp" and "global.psp" (this may require further changes to the chart).
```

Comment:
Wasn't this from other PR ? We can include here if its just cleanup. Want to make sure we aren't seeing a conflict.

Reply:
This is purposeful. I added more comments.
> Add fluentbit and fluentd resources to logging. Add clearer comments on
the two psp settings.